KTOR-9721 Swagger UI and OpenAPI docs need 3.1 compatibility updates (#839)
ktorio/ktor-documentation: topics/server-openapi.md

By default, documentation is rendered using `StaticHtml2Codegen`. You can customize the renderer inside the `openAPI {}`
block:

> > `StaticHtmlCodegen` and `StaticHtml2Codegen` are only safe for OpenAPI 3.0.x documents. OpenAPI 3.1 documents may

Reviewer: ```suggestion
> `StaticHtmlCodegen` and `StaticHtml2Codegen` support only OpenAPI 3.0.x documents. Using them with OpenAPI 3.1 documents may
```

> > fail to render correctly or produce incomplete HTML. For OpenAPI 3.1 browsing, prefer `swaggerUI` with Swagger UI

Reviewer: ```suggestion
> produce incomplete or incorrect HTML. For OpenAPI 3.1, use the [`swaggerUI()`](server-swagger-ui.md) function with Swagger UI
```
